### PR TITLE
429/Implemented unit test for filter checkbox component

### PR DESCRIPTION
**Pull request by mxrcury**
Unit tests for components/enchanced-table/fiter-checkbox

### CODE REVIEW

File: tests/unit/components/enhanced-table/filter-checkbox/FilterCheckbox.spec.jsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+describe('FilterCheckbox test', () => {
+  it('should add checkboxValue to filters when is checked', () => {
+    const { getByLabelText, rerender } = render(<FilterCheckbox filter={ mockedFilter } filterCheckbox={ mockedFilterCheckbox } setFilter={ mockedSetFilter } />)
+    
+    mockedCheckbox = getByLabelText('filter-checkbox')
+    expect(mockedCheckbox.checked).toBe(false)
+
+    fireEvent.click(mockedCheckbox)
+    mockedFilter.push(mockedFilterCheckbox.value)
+    rerender(<FilterCheckbox filter={ mockedFilter } filterCheckbox={ mockedFilterCheckbox } setFilter={ mockedSetFilter } />)
```
Comment (Tolik170) [inline tests/unit/components/enhanced-table/filter-checkbox/FilterCheckbox.spec.jsx:24]: do you need to rerender here?

File: tests/unit/components/enhanced-table/filter-checkbox/FilterCheckbox.spec.jsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+const mockedSetFilter = vi.fn()
+
+const mockedFilterCheckbox = {
+  label: 'adminTable.active',
+  value: 'active'
+}
+
+describe('FilterCheckbox test', () => {
+  it('should add checkboxValue to filters when is checked', () => {
+    const { getByLabelText, rerender } = render(<FilterCheckbox filter={ mockedFilter } filterCheckbox={ mockedFilterCheckbox } setFilter={ mockedSetFilter } />)
```
Comment (Tolik170) [inline tests/unit/components/enhanced-table/filter-checkbox/FilterCheckbox.spec.jsx:17]: try to render it once in beforeEach block and use screen.getByLabelText from '@testing-library/react'

File: tests/unit/components/enhanced-table/filter-checkbox/FilterCheckbox.spec.jsx (hunk 1 above)
Comment (mxrcury) [reply tests/unit/components/enhanced-table/filter-checkbox/FilterCheckbox.spec.jsx:24]: yes, cause i can't see checkbox changes

File: tests/unit/components/enhanced-table/filter-checkbox/FilterCheckbox.spec.jsx (hunk 2 above)
Comment (mxrcury) [reply tests/unit/components/enhanced-table/filter-checkbox/FilterCheckbox.spec.jsx:17]: i tried so but method getByLabelText imported from '@testing-library/react' didn't work properly and couldn't find checkbox by this label

Comment (mxrcury) [reply tests/unit/components/enhanced-table/filter-checkbox/FilterCheckbox.spec.jsx]: also entered prop filter in FilterCheckbox component have to be different in two test cases so i render it in every case

**Review by maarekua — APPROVED** [review]

**Review by Tolik170 — APPROVED** [review]

**Review by Yevhendzyuba — APPROVED** [review]

**Review by Yurenko — APPROVED** [review]

**Pull request merged**